File: js/src/test/Exchange/test.fetchTransfers.js
import testTransfer from './base/test.transfer.js';
import testSharedMethods from './base/test.sharedMethods.js';
async function testFetchTransfers(exchange, skippedProperties, code) {
    const method = 'fetchTransfers';
    const transfers = await exchange.fetchTransfers(code);
    testSharedMethods.assertNonEmtpyArray(exchange, skippedProperties, method, transfers, code);
    for (let i = 0; i < transfers.length; i++) {
        testTransfer(exchange, skippedProperties, method, transfers[i], code);
    }
    testSharedMethods.assertTimestampOrder(exchange, method, code, transfers);
    return true;
}
export default testFetchTransfers;

File: julia/Ccxt/test/exchange/test.fetchTransfers.jl
using Test
using Ccxt
function testFetchTransfers(exchange, skippedProperties, code)

    method = "fetchTransfers";
    transfers = Base.fetch(fetchTransfers(exchange, code));
    assertNonEmtpyArray(testSharedMethods, exchange, skippedProperties, method, transfers, code);
    i = 0
    while functions.ccxtruthy(functions.ccxt_lt(i, length(transfers)))
        testTransfer(exchange, skippedProperties, method, get(transfers, i + 1, nothing), code);
        i += 1
    end
    assertTimestampOrder(testSharedMethods, exchange, method, code, transfers);
    return true
end
